### PR TITLE
Improve keyring creation prompts and warnings

### PR DESCRIPTION
**Pull request by KPrasch**
**Type of PR:**
Feature

**Required reviews:** 
2

**What this does:**
Prompts the user with some additional info an detailed suggestions during password collection.

**Why it's needed:**
Helps to install an accurate sense of keystore diligence in anticipation of keyring portability.

### CODE REVIEW

File: nucypher/cli/painting/help.py [hunk 1]
```diff
@@ -55,7 +55,18 @@ def paint_new_installation_help(emitter, new_configuration, filepath):
...
-    emitter.message(f"Generated keyring {new_configuration.keyring_root}", color='green')
+    emitter.message(f"Generated keyring", color='green')
+    emitter.message(f"""
+    
+Public key (stamp):   {bytes(new_configuration.keyring.signing_public_key).hex()}
+Path to keyring: {new_configuration.keyring_root}
+
+- You can share your public key with anyone. Others need it to interact with you.
+- Never share secret keys with anyone! Character keys are required to interact with the network!
+- Backup your keyring! Without the keyring you wil not be able to use existing network policies.
```
Comment (fjarri) [inline nucypher/cli/painting/help.py:66]: typo: wil -> will

**Review by fjarri — APPROVED** [review]

File: nucypher/cli/painting/help.py (hunk 1 above)
Comment (derekpierre) [inline nucypher/cli/painting/help.py:66]: Is network policies too specific? What if we combine the Character keys statement with the backup bullet?

```suggestion
- Never share secret keys with anyone! 
- Backup your keyring! Character keys are required to interact with the protocol!
```

File: newsfragments/2662.misc.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Improves password collection hints while running `init` commands.
```
Comment (cygnusv) [inline newsfragments/2662.misc.rst:1]:
```suggestion
Improves password collection hints while running ``init`` commands.
```

**Review by cygnusv — APPROVED** [review]
A minor styling suggestion, but apart from that all good!

**Pull request merged**